DFE-Digital/sts-plan-technology-for-your-school
# Feat: updated C&S to improve vertical navigation

## Overview

Addresses ticket [230100](https://dfe-ssp.visualstudio.com/s190-schools-technology-services/_workitems/edit/230100)

Made improvements to C&S vertical navigation

## Changes

### Major

- Allowed for pages to choose which hero to use
- CSBodyTexts have their own slugs for use in vertical navigation
- Added unsupported element for when asset type is not recognised
- Removed unused home and privacy endpoints so the error page will display

## Release requirements

This release required a contentful update. 

- Need to add a slug, and useParentHero to the CSBodyText.

Probably easiest to use contentful merge to go from dev_test to staging_production when ready

![image](https://github.com/user-attachments/assets/d330654e-9d96-42ce-97d9-7a40997da1f6)


## How to review the PR

- Run the application and go to /content/information-asset-register
- Check that the overview page uses the default hero (Doesn't just say overview in the hero)
- Check all other steps use their own title and subtitle as the hero

## Checklist

Delete any rows that do not apply to the PR.
